src/vellum/workflows/nodes/displayable/prompt_deployment_node/node.py

```diff
@@ -37,6 +37,9 @@ class Outputs(BasePromptDeploymentNode.Outputs):
 
         text: str
 
+    def _validate(self) -> None:
+        pass
```

Comment:
[nit] no need to make this method abstract if downstream consumers are just going to skip anyway